### PR TITLE
Awesome deliverable

### PR DESCRIPTION
**Pull request by laboon**
This branch is awdsome!

### CODE REVIEW

**Comment by chuckie512** [comment]
Wow! that's awesome!

File: deliverables/awesome_deliverable.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+As part of this deliverable
+
+Be awesome
+
+Really awesome
+
+Super awesome
```
Comment (laboon) [inline deliverables/awesome_deliverable.md:7]: Really?  Three times using the word awesome?  What is this, 1995?

File: deliverables/awesome_deliverable.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Kitty cats
```
Comment (cfinest1) [inline deliverables/awesome_deliverable.md]: Kitty cats are not inherently awesome. Maybe if they were awesome kitty cats...

File: deliverables/awesome_deliverable.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+As part of this deliverable
+
```
Comment (laboon) [inline deliverables/awesome_deliverable.md:2]: Spaces are redundant algorithm

Comment (laboon) [inline deliverables/awesome_deliverable.md:2]: that doesn't even make sense

**Pull request merged**